ibis/backends/base/sqlglot/rewrites.py

```diff
@@ -104,7 +114,7 @@ def merge_select_select(_):
         if v.find((Window, ops.Unnest), filter=ops.Value):
             return _
     for v in _.predicates:
-        if v.find(ops.ExistsSubquery, filter=ops.Value):
+        if v.find(ops.Subquery, filter=ops.Value):
```

Comment:
What is the reason of this change?

Reply:
`InSubquery` has the same limitations as `ExistsSubquery` and without this change the generated code for tpch query 20 is incorrect.

This seems be an issue when using a projection for the leaf nodes (see `_tpch_table` for the Snowflake backend)